exonum/src/blockchain/service.rs

```diff
@@ -520,6 +520,6 @@ impl SharedNodeState {
 
 impl<'a, S: Service> From<S> for Box<dyn Service + 'a> {
     fn from(s: S) -> Self {
-        Box::new(s) as Box<dyn Service>
+        Box::new(s) as Self
```

Comment:
@DarkEld3r what do you think about cases like this?

Reply:
Honestly, I don't have a strong opinion here. 🤔